migrations/scripts/1_deploy_test.js

```diff
@@ -25,6 +19,23 @@ Enjoy the power of the Witnet Decentralized Oracle Network ;-)
     process.exit(1)
   }
   else {
-    console.log("Info: no migrations needed at this point.\n")
+    if (realm !== "default") {
+      if (
+        !settings.networks[realm] ||
+        !settings.networks[realm].test ||
+        !settings.networks[realm].test.network_id ||
+        settings.networks[realm].test.network_id == 4447 
+      ) {
+        console.error(`\nFatal: no "test" configuration found for ${realm.toUpperCase()} realm! Please, review 'migrations/erc2362.settings.js'.`)
+        process.exit(1)
+      }
+      let answer = prompt(`> Do you really want to run tests against ${realm.toUpperCase()} realm? [y/N] `).toLowerCase() 
```

Comment:
It is always a good idea to trim user input:
```suggestion
      let answer = prompt(`> Do you really want to run tests against ${realm.toUpperCase()} realm? [y/N] `).toLowerCase().trim() 
```